version.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <VersionPrefix>2.1.0</VersionPrefix>
+    <VersionSuffix>preview1</VersionSuffix>
+    <PackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(VersionSuffix)' == 'rtm' ">$(VersionPrefix)</PackageVersion>
```

Comment:
Wouldn't KoreBuild normally take care of this? Are we going to need this logic in every repo? I feel like doing this logic per-repo is going to lead to something being missed and us making bad versions at some point.

Reply:
Yes, this will be in each repo. Although currently the Universe build takes care of normalizing versions for "final" builds, the addition of aspnet/templating to aspnet/Universe, PCP and source-build efforts required that we also push this logic down into individual repos.